engine/security/https.md

```diff
@@ -82,13 +82,22 @@ using `10.10.10.20` and `127.0.0.1`:
 
     $ echo subjectAltName = DNS:$HOST,IP:10.10.10.20,IP:127.0.0.1 > extfile.cnf
 
+In any case, set the key's extended usage attributes, so if it is compromised
+it can't be used to connect to other dockerd's that know about it as a server:
+
+    $ echo extendedKeyUsage = serverAuth > extfile.cnf
+
     $ openssl x509 -req -days 365 -sha256 -in server.csr -CA ca.pem -CAkey ca-key.pem \
       -CAcreateserial -out server-cert.pem -extfile extfile.cnf
     Signature ok
     subject=/CN=your.host.com
     Getting CA Private Key
     Enter pass phrase for ca-key.pem:
 
+For more fine grained access control than certs (i.e. code that runs to
+determine whether a connection is allowed), use
+[AuthZ](../extend/plugins_authorization) plugins.
```

Comment:
we should mention that the authz plugins get the cert details.

"It is recommended to use [authorization plugins](../extend/plugins_authorization) for more fine-grained control to supplement authentication from mutual TLS. Authorization plugins running on a Docker daemon receive the certificate information for connecting Docker clients"

Reply:
Thanks, I fixed all these, changing your wording for the above slightly.